ts/utils/locale.ts

```diff
@@ -48,6 +48,9 @@ const preferredLanguageMappingToLocale = new Map<
   Locales
 >(Array.from(localeToPreferredLanguageMapping).map(item => [item[1], item[0]]));
 
+export const localeDateFormat = (date: Date, format: string): string =>
+  I18n.strftime(date, format);
```

Comment:
Could we add an extra check that the date is a valid object?
https://github.com/pagopa/io-app/blob/ea7218e99a37f2e4e5c2d51ca2a325fcfffd0cce/ts/utils/dates.ts#L18
If it is invalid it could raise an error